### PR TITLE
Htaccess for antora new folder

### PR DESCRIPTION
**Pull request by ebarboni**
Hi,
 trying to get antora ready if we wanna merge I think the htacess is the most mission critial file :p. And I cannot test it live.
 
 ns,dtds folder stay the same.

The files would be like that:
https://ci-builds.apache.org/job/Netbeans/job/netbeans-antora-website/lastSuccessfulBuild/artifact/site.zip
on the root folder there is a 10M of index for search lunr.

plugin, and update center redirect should stay the same.

The last part of redirect should handle all the former folder to new one. That's my expectations but I'm not sure. Handling the atom feed and former oracle netbeans folder to new antora folder.

### CODE REVIEW

File: supplemental-ui/.htaccess [hunk 1]
```diff
@@ -1,25 +1,10 @@
 DirectoryIndex index.html index.asciidoc
 
-# former path to antora path
-Redirect 301 /about/ /front/main/about/
```
Comment (vieiro) [inline supplemental-ui/.htaccess:4]: The current page in our website is https://netbeans.apache.org/about/. 
All search engines have already indexed that page, so when a user searchs Google and is directed to the "/about/" page we want to behave nicely to them, this is, we do want to redirect them to "/front/main/about". So I would keep this one.

File: supplemental-ui/.htaccess [hunk 2]
```diff
@@ -1,25 +1,10 @@
...
-Redirect 301 /blogs/ /front/main/blogs/
-Redirect 301 /jackpot/ /front/main/jackpot/
-Redirect 301 /community/ /front/main/community/
-Redirect 301 /help/ /front/main/help/
-Redirect 301 /projects/ /front/main/projects/
-Redirect 301 /participate/ /front/main/participate/
-Redirect 301 /kb/ /tutorial/main/kb/
-Redirect 301 /tutorials/ /tutorial/main/tutorial/
-Redirect 301 /wiki/ /wiki/main/wiki/
```
Comment (vieiro) [inline supplemental-ui/.htaccess:13]: Same thing for all of these. If a user is sent to "/wiki" from a search engine, we want to redirect the users to "/wiki/main/wiki" instead. So we want to keep these redirections.

File: supplemental-ui/.htaccess [hunk 3]
```diff
@@ -1,25 +1,10 @@
...
-
-
+# download from incubation
 Redirect 302 /download/9.0-beta/source https://archive.apache.org/dist/incubator/netbeans/incubating-netbeans-java/incubating-9.0-beta/incubating-netbeans-java-9.0-beta-source.zip
 Redirect 302 /download/9.0-beta/binary https://archive.apache.org/dist/incubator/netbeans/incubating-netbeans-java/incubating-9.0-beta/incubating-netbeans-java-9.0-beta-bin.zip
 
-# former path to antora path after letting download to 
-Redirect 301 /download/ /front/main/download/
```
Comment (vieiro) [inline supplemental-ui/.htaccess:20]: Keep this one too.

File: supplemental-ui/.htaccess [hunk 4]
```diff
@@ -92,6 +77,20 @@ RedirectMatch 301 ^/.?downloads/.*$ /download/index.html
 # Redirect plugin page to plugin portal
 Redirect 302 /plugins/index.html https://plugins.netbeans.apache.org/
 
+# former path to antora path after letting download to 
+Redirect 301 /about/ /front/main/about/
```
Comment (vieiro) [inline supplemental-ui/.htaccess:81]: Ah, I see you keep the redirections here. Why the change from the top of the file to the bottom of the file? Is that important (order or redirections is important in .htaccess files, AFAIR).

**Comment by vieiro** [comment]
Hi, I'm unsure about the changes in the .htaccess files. I need some clarifications!

**Comment by ebarboni** [comment]
It's to let the right url take priority and trying to cover all former path into the new

**Review by vieiro — APPROVED** [review]
Ok. Let's try it out then.
